### PR TITLE
EDU: Update 1995 content to match DEPO styles #13965

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/13965

This PR covers AC1,2,3,5,6

## Screenshots
Screen Shot 2020-10-05 at 11 38 44 AM
Screen Shot 2020-10-05 at 11 39 24 AM
Screen Shot 2020-10-05 at 11 40 14 AM
Screen Shot 2020-10-05 at 11 40 50 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
lgtm

**Review by rianfowler — APPROVED** [review]
We can probably update the codeowner on this so it doesn't require a review by the frontend-review-group

**Pull request merged**
